core/src/main/java/io/grpc/internal/SpiffeUtil.java

```diff
@@ -119,4 +286,34 @@ public String getPath() {
     }
   }
 
+  /**
+   * Represents a Trust Bundle inspired by SPIFFE Bundle Format standard. Only trust domain's
```

Comment:
Suggested edit for clarity: Represents a SPIFFE trust bundle; that is, a map from trust domain to set of trusted certificates.